components/admin/data/layers/table/component.js

```diff
@@ -36,51 +37,29 @@ class LayersTable extends PureComponent {
     pagination: INITIAL_PAGINATION,
     loading: true,
     layers: [],
-    filters: { name: null }
+    filters: { name: null, 'user.role': 'ADMIN' }
   }
 
   componentDidMount() {
```

Comment:
Can we change it to `componentWillMount`? This way the request will fetch sooner.